test/gui/shared/steps/steps.py

```diff
@@ -239,6 +239,20 @@ def step(context, receiver, resource, permissions):
     openSharingDialog(context, resource)
     shareItem = SharingDialog()
     shareItem.addCollaborator(receiver, permissions)
+    clickButton(waitForObject(shareItem.SHARING_DIALOG_CLOSE_BUTTON))
```

Comment:
why did you move it from Page object to the steps?

Reply:
Because we only need this in few place not for all `addCollaborator`. So I thought it would be better to have it in here.